### DISCUSSION TITLE
Plex Doesn't See Chapters

### DISCUSSION THREAD

**Discussion by apurgert** [seanap/Plex-Audiobook-Guide#92] [Q&A] [answered]
I converted multiple MP3's into one chapterized M4B file via AudiobookConverter, then ran it through Mp3tag.  When running in VLC, I can see the chapters, skip back and forth without issue, etc.  

When I move it to Plex, it only sees the M4B as one long track and the next chapter button >| is greyed out.

Is there a different way to tell Plex about the chapters?

**Comment by seanap** [comment, marked as answer]
Plex doesn't officially support audiobooks, so the chapters for an m4b don't display in Plex.  To get the chapters to display you'll need to use a 3rd party player that connects to your plex server like Prologue, or Bookcamp.

AudiobookShelf is another self hosted server (alternative to Plex) which has it's own player app and both the server and player display the chapters.

**Reply by apurgert** [reply]
Thanks for the explanation!  I will take a look at those.  Great write up by the way, thanks for putting this together!

**Reply by seanap** [reply]
Thanks! If you have an iphone Plex+Prologue is amazing.  For android Audiobookshelf would probably be the best option for now, it's what I've been using. Also with ABS it's really easy to browse your server and dl books locally to your phone, then you can point any local audiobook app (eg. Smart, Listen, Voice, etc) to your local storage. I end up just using the ABS app on my android.

I just wish PlexAmp would buy Prologue and make it cross platform, PlexAmp is great for browsing, sorting, filtering, creating custom smart collections, but it's terrible as an audiobook player.

At the end of the day with properly tagged and organized books you can use any server/client/app you want, and it's easy to switch to a new platform if something comes out.
